Update msiv1_token_revocation.md to specify MIRP api-version (#5202)
AzureAD/microsoft-authentication-library-for-dotnet: docs/msiv1_token_revocation.md

6. **MITS** is basically a proxy, forwarding the query to **SFRP**.
7. **SFRP** uses MSAL again to get a **new** token from eSTS.

> [!IMPORTANT]

> > This design is only applicable to MIRP api-version=2025-03-30 (for App Service). api-version for service fabric will be soon made available. 

Reviewer: Chatted with SF folks before. They do not intend to bump their API version, so, it will remain unchanged (as 2019-07-01-preview).

Reviewer: Well, I meant to suggest a modification.
```suggestion
> This design is only applicable to MIRP api-version=2025-03-30 (for App Service). api-version for service fabric will remain unchanged as `2019-07-01-preview`. 
```

If you are seeking confirmation from SF, we should have not merged this PR until their confirmation comes back.
